<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttendanceForm</class>
 <widget class="QWidget" name="AttendanceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mark Attendance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_employee">
     <property name="text">
      <string>Select Employee ID:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_date">
     <property name="text">
      <string>Select Date:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dateEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>Attendance Status:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Present</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Absent</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_markAttendance">
     <property name="text">
      <string>Mark Attendance</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
